modules/kernel/src/main/java/org/opencastproject/kernel/security/AuthenticationSuccessHandler.java

```diff
@@ -63,7 +62,12 @@
    */
   @Override
   public void onAuthenticationSuccess(HttpServletRequest request, HttpServletResponse response,
-          Authentication authentication) throws IOException, ServletException {
+          Authentication authentication) throws IOException {
+
+    if ("application/json".equalsIgnoreCase(request.getHeader("Accept"))) {
```

Comment:
Perhaps I'm unclear on what exactly the Accept header can contain, but can't it also be a list?  What if it's 'application/json,application/xml'?

Reply:
Doesn't this also mean that *any* successful request with an accept for json gets redirected to `info/me.json`, or am I misreading what this class does?

Reply:
But right now only if it's specifically `application/json`. If it's JSON or a web page, they will get the web page

Reply:
It doesn't seem totally intuitive to me to provide a redirect to an application/json request rather than say a 200 and a json response in the body like {"status":"success"}.